### ISSUE TITLE
CSS fails to load with some pages

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-html5-windows#1]
See https://github.com/kiwix/kiwix-html5/issues/149

**Comment by Jaifroid** [comment]
OK, I've found the problem. See https://github.com/kiwix/kiwix-html5/issues/253 and have a potential solution:
```
//var TOTAL_MEMORY = Module["TOTAL_MEMORY"] || 8e7; //- GK not enough startup memory
var TOTAL_MEMORY = Module["TOTAL_MEMORY"] || 14000000;
```
in xzdec.js.

**Comment by Jaifroid** [comment]
Closing, as cause of failure to load CSS has been found and implemented in PR #6 .
